src/NuGet.Core/NuGet.Protocol/Utility/ValidatePackageId.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using NuGet.Common;
+
+namespace NuGet.Protocol
+{
+    internal class ValidatePackageId
```

Comment:
>ValidatePackageId [](http://example.com/codeflow?start=19&length=17)

Would it make more sense to name this class PackageIdValidator? That way, calls would look like PackageIdValidator.Validate()? Also, is there is a reason why this class shouldn't be static?